### ISSUE TITLE
Add integration test for 'minishift profile set' for a non existing profile 

### ISSUE THREAD

**Issue by anjannath** [minishift/minishift#1577]
Integration test to check the behaviour of `minishift profile set`, as the new update to the command will create a profile and set it as active if it did not exist earlier.

**Comment by LalatenduMohanty** [comment]
@anjannath we have an integration test covering the `profile set` for existing instance as below but we do not have an scenario when we are running `profile set` for non-existent profile instance. 
```
Scenario: User should be able set 'minishift' as the active profile
    When executing "minishift profile set minishift" succeeds
     Then stdout should contain
     """
     Profile 'minishift' set as active profile
     """
```

**Comment by LalatenduMohanty** [comment]
@anjannath This is getting automatically fixed when I am fixing #1574 . See PR https://github.com/minishift/minishift/pull/1578

**Comment by anjannath** [comment]
@LalatenduMohanty Cool.. this can be closed then.
Resolve by #1578

**Comment by LalatenduMohanty** [comment]
@anjannath re-opening it as it is a valid issue and it getting fixed as part of #1578.

**Comment by LalatenduMohanty** [comment]
Resolved via https://github.com/minishift/minishift/pull/1578/
